gap/FinSetsForCAP.gi

```diff
@@ -685,6 +685,30 @@ AddIsSplitMonomorphism( FinSets,
     return IsInitial( Range( phi ) ) or ( not IsInitial( Source( phi ) ) and IsMonomorphism( phi ) );
 end );
 
+##
+AddIsLiftable( FinSets,
+  function ( beta, alpha )
+    
+    return IsSubset( AsList( ImageObject( alpha ) ), AsList( ImageObject( beta ) ) );
+    
+end );
+
+##
+AddLift( FinSets,
+  function ( beta, alpha )
+    local L, imgs, chi;
+    
+    L := AsList( Source( beta ) );
+    
+    imgs := List( L, beta ); ## subset of N = Range( beta ) = Range( alpha )
```

Comment:
This is a very expensive way of more or less reconstructing `AsList( beta )`, or am I missing something?

Reply:
Good idea. See new push.